lrpar/src/lib/parser.rs

```diff
@@ -907,4 +907,30 @@ Call: 'ID' '(' ')';";
         let err_tok_id = usize::from(grm.token_idx("ID").unwrap()).to_u16().unwrap();
         assert_eq!(errs[0].lexeme(), &Lexeme::new(err_tok_id, 2, 1));
     }
+
+    #[test]
+    fn test_actions() {
+        // From p4 of https://www.cs.umd.edu/class/spring2014/cmsc430/lectures/lec07.pdf
+        check_parse_output(
```

Comment:
Don't you need to check the output of this function? Maybe at least doing an `unwrap()`?